.idea/compiler.xml

```diff
@@ -2,6 +2,6 @@
 <project version="4">
   <component name="TypeScriptCompiler">
     <option name="recompileOnChanges" value="true" />
-    <option name="typeScriptCompilerParams" value="--noEmit --target es5 --jsx react" />
+    <option name="typeScriptCompilerParams" value="--noEmit --target es5 --jsx react --esModuleInterop true" />
```

Comment:
WebStorm doesn't seem to like to read the tsconfig.json and looks at its own .idea file. Copied more of that in here to make it work properly.